etc/ci/buildbot_steps.yml

```diff
@@ -68,6 +68,7 @@ linux-rel-nogate:
   - ./mach clean-nightlies --keep 3 --force
   - ./mach build --release
   - python ./etc/ci/chaos_monkey_test.py
+  - python ./etc/ci/mutation_test.py
```

Comment:
Not sure we want to gate based on mutation test results. Should we run this nightly instead?

Reply:
I am not sure about this. @jdm suggested me to add it to linux-rel-nogate. Because "It does not cause the entire CI run to fail if any of the tests fail. This is useful for running interesting, randomized tests on merged PR but reviewing the results at a later time."

Reply:
Sounds sensible. If linux-rel-nogate is run on every PR, we should do very few rounds of testing each time. Building and testing takes long enough as it is!